OOP Fundamentals and C#/09 - Design Patterns - Creational/Singleton/Program.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Singleton;
+
```

Comment:
Could we also add a static constructor version of singleton, please

Reply:
added example using static